design/sketch.webidl

```diff
@@ -453,13 +455,14 @@ interface WebGPUCommandEncoder {
 
 // Fence
 interface WebGPUFence {
-    u32 getValue();
+    bool wait(u64 timeout);
+    Promise then();
```

Comment:
nits: `bool`->`boolean`; `Promise then()`->something like [this](https://github.com/gpuweb/gpuweb/pull/49/files#diff-7d154c8abbb1fb80041e213976224980R15)